decidim-elections/app/commands/decidim/elections/admin/add_user_as_trustee.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Elections
+    module Admin
+      # This command is executed when the user creates a Trustee
+      # from the admin panel.
+      class AddUserAsTrustee < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # form - A form object with the params.
+        def initialize(form, current_user)
+          @form = form
+          @current_user = current_user
+        end
+
+        # Creates the trustee if valid.
+        #
+        # Broadcasts :ok if successful, :invalid otherwise.
+        def call
+          return broadcast(:invalid) if form.invalid?
+          return broadcast(:exists) if existing_trustee_participatory_spaces?
+
+          transaction do
+            notify_user_about_trustee_role if new_trustee?
```

Comment:
I think this could be problematic, because you notify the user after completing the transaction. Also, you probably don't want to try to create a new trustee if there is already a trustee for that user. 

I'd move this notification to the end of the transaction, like this:

```ruby
transaction do
  add_user_as_trustee! if new_trustee?
  add_participatory_space
  notify_user_about_trustee_role if new_trustee?
end
```